Assets/Scripts/UIProgress.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using UnityEngine;
+using UnityEngine.UI;
+using Debug = UnityEngine.Debug;
+
+public class UIProgress : MonoBehaviour
```

Comment:
Find a better name for this, the class is mapping a real world position progress in a UI slider, so the name should indicate that